javav2/usecases/Creating_serverless_aurora_app/Readme.md

```diff
@@ -42,9 +51,9 @@ To complete the tutorial, you need the following:
 
 ### Creating the resources
 
-Create an Amazon Serverless Amazon Aurora database named **jobs**. For information, see [Creating an Aurora Serverless v1 DB cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.create.html).
+Create an Amazon Serverless Amazon Aurora database named **jobs**. For more information, see [Creating an Aurora Serverless v1 DB cluster](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless.create.html).
 
-To successfully connect to the database using the **RdsDataClient** object, you must setup an AWS Secrets Manager secret that is used for authentication. For information, see [Rotate Amazon RDS database credentials automatically with AWS Secrets Manager](https://aws.amazon.com/blogs/security/rotate-amazon-rds-database-credentials-automatically-with-aws-secrets-manager/). 
+To successfully connect to the database using the **RdsDataClient** object, set up an AWS Secrets Manager secret that is used for authentication. For more information, see [Rotate Amazon RDS database credentials automatically with AWS Secrets Manager](https://aws.amazon.com/blogs/security/rotate-amazon-rds-database-credentials-automatically-with-aws-secrets-manager/). 
 
 To use the **RdsDataClient** object, you require these two ARN values: 
```

Comment:
To use the **RdsDataClient** object, you require the following two Amazon Resource Name (ARN) values:

Reply:
fixed